packages/larva-css/src/02-algorithms/a-carousel-grid/a-carousel-grid.common.inline.scss

```diff
@@ -0,0 +1,63 @@
+@import 'setup';
+
+.lrv-a-carousel-grid {
```

Comment:
Refactor to use custom properties to contain the values that change with different variations e.g. https://codepen.io/laras126/pen/bGWLbwY

The ideal is to update the layout by adding a single class `lrv-a-carousel-grid--overlay` and the property values are updated with custom properties so no specificity is added.

Also see https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva-css/src/02-algorithms/a-glue/a-glue.common.inline.scss